package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "casbin-prisma-adapter",
-  "version": "1.1.3",
+  "version": "1.2.0",
```

Comment:
I don't think you should update versions in this file manually

Reply:
It changed when I ran `npm i`.. I'll take another look. Thanks!